src/Bigcommerce/Api/Resource.php

```diff
@@ -41,7 +46,8 @@ public function __construct($object=false)
 
 	public function __get($field)
 	{
-		if (method_exists($this, $field) && isset($this->fields->$field)) {
+		$fieldName = isset($this->fieldMap[$field]) ? $this->fieldMap[$field] : $field;
+		if (method_exists($this, $field) && isset($this->fields->$fieldName)) {
```

Comment:
seemed weird when I first read this hah, maybe we could add a comment saying ``// call the child class's method for the field being accessed if defined` or something?